packages/snaps-controllers/src/interface/utils.ts

```diff
@@ -17,14 +18,23 @@ import type {
   RadioElement,
   SelectorElement,
   SelectorOptionElement,
+  AccountSelectorElement,
 } from '@metamask/snaps-sdk/jsx';
 import { isJSXElementUnsafe } from '@metamask/snaps-sdk/jsx';
 import {
+  createAddressList,
   getJsonSizeUnsafe,
   getJsxChildren,
   getJsxElementFromComponent,
   walkJsx,
 } from '@metamask/snaps-utils';
+import { parseCaipAccountId, type CaipAccountAddress } from '@metamask/utils';
+
+type GetSelectedAccount = () => InternalAccount | undefined;
+type GetAccountByAddress = (
+  address: CaipAccountAddress,
+) => InternalAccount | undefined;
+type SetSelectedAccount = (accountId: string) => void;
```

Comment:
Nit for readability:

```suggestion
type GetSelectedAccount = () => InternalAccount | undefined;

type GetAccountByAddress = (
  address: CaipAccountAddress,
) => InternalAccount | undefined;

type SetSelectedAccount = (accountId: string) => void;
```

Maybe add JSDoc too.